SeaPublicWebsite/Controllers/EnergyEfficiencyController.cs

```diff
@@ -1143,7 +1144,15 @@ public async Task<IActionResult> YourRecommendations_Post(YourRecommendationsVie
 
             if (viewModel.HasEmailAddress == HasEmailAddress.Yes)
             {
-                emailApi.SendReferenceNumberEmail(userDataModel.EmailAddress, userDataModel.Reference);
+                try
+                {
+                    emailApi.SendReferenceNumberEmail(userDataModel.EmailAddress, userDataModel.Reference);
+                }
+                catch (NotifyClientException)
+                {
+                    ModelState.AddModelError(nameof(viewModel.EmailAddress), "Enter a valid email address");

```

Comment:
I suspect that this isn't the only reason that we could get an error from the Notify API so this could be quite annoying to the user if actually they've entered a valid email and the Notify API is down for other reasons.

Reply:
Hmm, that might be true. I'll add some error messages when it's not the user's fault.